docs/data/stashcache/install-cache.md

```diff
@@ -108,89 +138,84 @@ and runs alongside the unauthenticated instance.
+
 
 ### Enable remote debugging
 
-This feature enables remote debugging via the `digFS` read-only file system, it's optional line in the config file that was created when configuring the cache:
+XRootD provides remote debugging via a read-only file system named digFS.
+This feature is disabled by default, but you can enable it when you need assistance with your server.
 
-```
-xrootd.diglib * /etc/xrootd/digauth.cf
-```
+To enable remote debugging, edit `/etc/xrootd/digauth.cfg` and specify the authorizations for reading digFS.
 
-where `/etc/xrootd/digauth.cf` may have following content:
+See [the XRootD manual](http://xrootd.org/doc/dev48/xrd_config.htm#_Toc496911334) for the syntax.
 
-```
-all allow host h=abc.org
-all allow host h=*.xyz.edu
-```
+Remote debugging should only be enabled for as long as you need assistance.
+Once your problem has been resolved, turn it off.
 
-## Managing StashCache and associated services
 
-StashCache daemons are managed by systemd units.  First ensure that your cache directory (default `/stash`) is
-mounted, then ensure you *enable* (set to start at boot) and *start* the StashCache-related services.
+Managing StashCache and associated services
+-------------------------------------------
 
+These services must be managed by `systemctl` and may start additional services as dependencies.
 As a reminder, here are common service commands (all run as `root`) for EL7:
 
 | To...                                   | On EL7, run the command...         |
 | :-------------------------------------- | :--------------------------------- |
 | Start a service                         | `systemctl start <SERVICE-NAME>`   |
-| Stop a  service                         | `systemctl stop <SERVICE-NAME>`    |
+| Stop a service                          | `systemctl stop <SERVICE-NAME>`    |
 | Enable a service to start on boot       | `systemctl enable <SERVICE-NAME>`  |
 | Disable a service from starting on boot | `systemctl disable <SERVICE-NAME>` |
 
-### Public Cache Services
+### Public cache services
+
 | **Software** | **Service name** | **Notes** |
 |--------------|------------------|-----------|
-| XRootD | `xrootd@stashcache-cache-server.service` | The xrootd daemon, which performs the data transfers |
-| StashCache Reporter | `stashcache-reporter.timer` | Report cache statistics to central OSG collector |
+| XRootD | `xrootd@stash-cache.service` | The xrootd daemon, which performs the data transfers |
 | Fetch CRL | `fetch-crl-boot` and `fetch-crl-cron` | Required to authenticate monitoring services.  See [CA documentation](/common/ca#managing-fetch-crl-services) for more info |
 
-### Authenticated Cache Services
 
-_In addition_ to the public cache services, there are three systemd units specific to the authenticated cache.
+### Authenticated cache services
+
+In addition to the public cache services, there are three systemd units specific to the authenticated cache.
```

Comment:
We should add a qualifier saying that they only need to do this if they configured an auth cache (and link to the config section)

Reply:
It says "(optional)" in the header 2 lines up...